src/pyaro/timeseries/Filter.py

```diff
@@ -579,16 +579,15 @@ def init_kwargs(self):
         }
 
     def _index_from_include_exclude(self, times1, times2, includes, excludes):
-        idx = times1.astype("bool")
         if len(includes) == 0:
-            idx[:] = True
+            idx = np.repeat(True, len(times1))
         else:
-            idx[:] = False
+            idx = np.repeat(False, len(times1))
             for start, end in includes:
-                idx |= (start <= times1) & (times2 <= end)
+                idx |= (np.datetime64(start) <= times1) & (times2 <= np.datetime64(end))
 
```

Comment:
I'm not sure why the conversion to np.datetime64 is needed here, or, more specific, if operations need to be done as np.datetime64 then maybe _str_list_to_datetime_list should do the conversion already?

Reply:
Probably should store dt's as numpy format, opened issue #59 for this